src/Collection.php

```
@@ -354,11 +410,22 @@ public function drop()
      * @see http://docs.mongodb.org/manual/reference/method/db.collection.dropIndex/
      * @param string $indexName
      * @return Cursor
-     * @throws InvalidArgumentException if "*" is specified
+     * @throws InvalidArgumentException if "*" is specified, since dropIndexes()
+     *                                  should be used to drop multiple indexes
      */
     public function dropIndex($indexName)
     {
-        // TODO
+        // TODO: Add support for dropping indexes by keys/options (e.g. IndexModel)
+        $indexName = (string) $indexName;
```

If this string is empty, we should raise an InvalidArgumentException.